package-lock.json

```diff
@@ -1,3820 +1,8 @@
 {
   "name": "chartjs-adapter-luxon",
   "version": "1.0.0",
-  "lockfileVersion": 2,
```

Comment:
can you use npm 7 and then run `npm install`? I don't want to switch back and forth between lockfileVersions every commit

Reply:
@benmccann Updated